### DISCUSSION TITLE
A site built with Perplex, for the interested

### DISCUSSION THREAD

**Discussion by danielfdickinson** [bowman2001/perplexdoc#25] [Show and tell]
I've re-published <https://www.princesandmadmen.ca> using Perplex. Feel free to visit and report an issue or use the contact form if you want to reach me.

I hope you find it was worth your time if you have a peek!

**Comment by bowman2001** [comment]
I’m glad you like the theme and could re-deploy your blog so quickly!

I can still see a few issues where the layout could be improved:

- Featured SVGs don’t fit into the card layout: https://github.com/bowman2001/perplex/issues/140
- The spacing between the blog title and the text is too small. Already taken care of in the latest theme version.
- The portrait image is elliptical and cropped. You could add some padding to the original and make it quadratic to let it fit better into the intended circle. (updated the docs accordingly)

And I have a personal request: Please remove the author page for me from your blog site. I’m only responsible for the theme layout and not your content. You are giving enough credit by presenting the theme logo in the footer and by making contributions to its development.

**Reply by danielfdickinson** [reply]
For the last: Will do, meant to ask about that. Thanks!

And thank you for looking at it and making updates.

**Reply by danielfdickinson** [reply]
Update deployed with an author page for you removed. I appreciate all you do, and wanted to give credit where credit was due, but appreciate the potential for confusion. Thank you for noticing and letting me know what you wanted.

**Comment by danielfdickinson** [comment]
Speaking of the spacing; I may have missed some CSS updates; I had some CSS at the bottom of a local copy of screen.css, but that risks missing your updates. In addition, the CSS I've added doesn't play well with PostCSS (I have attribute selectors, which get removed by the purge and break my intended changes). So, I'll update the css.html to look for additional CSS under css/custom and add it is a second stylesheet, if it exists (and will avoid PostCSS for the custom CSS).

FYI, the purpose of the CSS is to invert SVGs that have transparent background, but dark (usually black) lines and thus have quite low contrast in dark mode. Not sure if I've handled it the best way, but it works for now.

The CSS snippet is this:

```css
@media (prefers-color-scheme: dark) {
  figure[id^="dktp"] img,
  figure[id^="dktp"] svg {
    filter: invert(1);
  }
}
```

**Comment by danielfdickinson** [comment]
While we're here: I haven't figured out how to resolve the small amount of space between the blog section title and a box as shown here:

![image](https://github.com/bowman2001/perplexdoc/assets/20735818/fdc8f03c-cfd5-4cfc-b112-b9c1512d40e4)

Is there a way I can adjust that?

**Comment by danielfdickinson** [comment]
I've made a small number of customisations I'd be happy to PR if you are interested:

1. A contact form shortcode that works with Netlify or a provider such as as Formspree.io (which I use)
2. A small amount of CSS for the contact form
3. For those using Apache an .htaccess generator for aliases (set `disableAliases: true` to disable the Hugo meta refresh aliases and have real 301 redirect headers using `mod_alias` in Apache, instead). One can also add arbitrary redirects with the partial I use.

**Comment by danielfdickinson** [comment]
FYI https://www.danielfdickinson.ca is now live with Perplex (with some tweaks / fixes that I plan to send your way).
